app/src/main/java/com/jasminsp/weatherapp/MainActivity.kt

```diff
@@ -47,9 +60,12 @@ class MainActivity : ComponentActivity(), SensorEventListener {
                     modifier = Modifier.fillMaxSize(),
                     color = MaterialTheme.colors.background
                 ) {
+                    NavHost(navController, startDestination = "main view") {
+                        composable("main view") { MainView(navController) } // Replace with reference to official Composable
+                        composable("my location") { MyLocation(navController) } // Replace with reference to official Composable
```

Comment:
Route names would probably be better without the gap to remain the consistency off the app being camelCased.

Reply:
It's common to have the navigation names as "normal" text as they are not variables or functions. I looked into it a bit too.

Reply:
Yes technically it’s not a problem but in my opinion when we start using routes with arguments, it is clearer to use myLocation/id. It could be preference too but this would follow the style we have used in the naming in general😊 Also one thing I was thinking to have a constants.kt file where we can define texts like this and then just call the constant such as routeMain or routeMyLocation. This way we avoid hard coding anything in the code itself.